File: Leetcode/javascript/richest-customer-wealth.js
// https://leetcode.com/problems/richest-customer-wealth/submissions/

var maximumWealth = function(accounts) {
    console.log(accounts)
    
    var largestSum = 0
    for (i = 0; i < accounts.length; i++){
        var iList = accounts[i]
        console.log(iList)
        
       var listSum = 0
        for (j = 0; j < iList.length; j++) {
            listSum += iList[j]
            console.log(listSum)
        }   
        
        if (listSum > largestSum) {
            largestSum = listSum
        }   
    }    
    return largestSum
 
};

File: Leetcode/python/richest-customer-wealth.py
# https://leetcode.com/problems/richest-customer-wealth/submissions/

def maximumWealth(accounts):
    print(accounts)
    
    largestSum = 0
    for i in range(len(accounts)):
        iList = accounts[i]
        print(iList)
        
        listSum = 0
        for num in iList:
            listSum += num
        print(listSum)
        
        if listSum > largestSum:
            largestSum = listSum
    
    return largestSum